Tools/autotest/param_metadata/param_parse.py

```diff
@@ -89,17 +109,13 @@ def find_vehicle_parameter_filepath(vehicle_name):
 setattr(ap_vehicle_lib, "Path", os.path.join('..', 'libraries', 'AP_Vehicle', 'AP_Vehicle.cpp'))
 libraries.append(ap_vehicle_lib)
 
+libraries.append(lua_applets())
+
 error_count = 0
 current_param = None
 current_file = None
 
 
-def debug(str_to_print):
```

Comment:
Moved for some reason?

Reply:
yes, used in an earlier place, needed to move